sqlglot/transforms.py

```diff
@@ -466,6 +466,91 @@ def unqualify_columns(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
+def remove_unique_constraints(expression: exp.Expression) -> exp.Expression:
+    assert isinstance(expression, exp.Create)
```

Comment:
should we instead switch this to `if isinstance(...):` and then return expression by default as in other transformations?

Reply:
Probably better to fail. If this isn't exp.Create, something has gone wrong.
I'd set it as the type annotation.. but the typing is hard.